triton-docker

```diff
@@ -68,11 +68,28 @@ function currentProfile () {
         local profile_source='default; use `triton profile set <profile name>` to change'
     fi
 
-    echo "Executing in '$(triton profile get | awk 'NR == 1 {print $2}')' (${profile_source})"
+    # Get and the user's account information and CloudAPI URL for parsing later
+    local triton_account="$(triton account get)"
+
+    echo "Executing in '$(echo "${triton_account}" | awk 'NR == 1 {print $2}')' (${profile_source})"
+
+    # Check if CNS is enabled, require it
+    if [ ! "true" == "$(echo "${triton_account}" | awk -F": " '/cns/{print $2}')" ]
+    then
+        echo
+        tput rev  # reverse
+        tput bold # bold
+        echo 'Error! Triton CNS is not enabled.'
+        tput sgr0 # clear
+        echo
+        echo 'Please run the following to activate:'
+        echo '  triton account update triton_cns_enabled=true'
+        echo
+    fi
 }
```

Comment:
You want this to be an error? Rather than a warning?

If an *error* it should exit non-zero.